client/src/AppBundle/Resources/translations/report-pa-fee-expense.en.yml

```diff
@@ -29,49 +29,46 @@ feeExistPage:
       label: Save and continue
 
 editPage:
-  htmlTitle: "Deputy report - Deputy fees and expenses | GOV.UK"
+  htmlTitle: "Deputy fees and expenses - Complete the deputy report | GOV.UK"
   pageTitle: Deputy fees and expenses
   pageSectionDescription: Please tell us about any fees or expenses you've claimed from the client's funds in line with the practice direction.
 
-
 otherExistPage:
-  htmlTitle: "Deputy report - Deputy fees and expenses  | GOV.UK"
+  htmlTitle: "Deputy fees and expenses - Complete the deputy report | GOV.UK"
   pageTitle: Deputy fees and expenses
   form:
     paidForAnything:
       label: Have you claimed any other expenses or fees outside of the practice direction?
     save:
       label: Save and continue
 
-
-
 otherAddPage:
-  htmlTitle: "Deputy report - Deputy fees and expenses | GOV.UK"
+  htmlTitle: "Add an expense - Complete the deputy report | GOV.UK"
   pageTitle: Add an expense
 
 otherAddAnotherPage:
-  htmlTitle: "Deputy report - Deputy fees and expenses | GOV.UK"
+  htmlTitle: "Add an expense - Complete the deputy report | GOV.UK"
   pageTitle: Add an expense
   form:
     addAnother:
       label: The expense has been saved. Would you like to add another now?
       hint: If you prefer, you can come back to this section later to add more expenses
     save:
-       label: Continue
+      label: Continue
 
 otherEditPage:
-  htmlTitle: "Deputy report - Deputy fees and expenses | GOV.UK"
+  htmlTitle: "Edit expense - Complete the deputy report | GOV.UK"
   pageTitle: Edit expense
 
 summaryPage:
-  htmlTitle: "Deputy report -  Deputy fees and expenses | GOV.UK"
-  pageTitle:  Deputy fees and expenses
+  htmlTitle: "Deputy fees and expenses - Complete the deputy report | GOV.UK"
+  pageTitle: Deputy fees and expenses
   pageSectionDescription: |
     In this section, we asked you to tell us about any fees or expenses you have charged in line with the practice direction.
   addButton: Add a deputy expense
 
 deletePage:
-  htmlTitle: "Deputy report - Deputy fees and expenses | GOV.UK"
+  htmlTitle: "Deputy fees and expenses - Complete the deputy report | GOV.UK"
```

Comment:
Remove fees and expenses?

Reply:
Believe this relates to the subtitle and when viewing the page myself it makes sense to have the fees and expenses in the title